armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -800,3 +806,129 @@ def isTargetComponent(self, c):
             Component to check target component status
         """
         return bool(c in self._componentDeterminesBlockHeight)
+
+
+class AxialExpMassConservationSummaryLogger:
+    """Class to log and print axial expansion mass conservation."""
+
+    # do not do logging for verbosity levels greater than or equal to
+    _LOGGING_VERBOSITY_THRESHOLD = 20  # runLog.info
```

Comment:
I think you can pull this value straight from the `runLog` module (or the `logging` module which it inherits from) in order to keep those consistent.

Reply:
@john-science turns out that this approach is sensitive to parallelization and doesn't really work with our current testing environment.... particularly in regard to the global cs struggles.... 

It'd be great to be able to populate these summary reports based on the runLog verbosity, and in serial, I think we can. But since we struggle with parallelization in regard to changing case settings, I can't fiddle my way to get the tests to pass in parallel. 

Any suggestions would be welcomed!

Reply:
```python
from logging import INFO
```

Or:

```python
import logging

logging.INFO
```

Reply:
https://github.com/terrapower/armi/pull/1338/commits/9d8362b40d2be61277e8a43210f26636c1a8f785